### ISSUE TITLE
Training details?

### ISSUE THREAD

**Issue by JiaaiLiu** [auspicious3000/autovc#37]
I am training your model with one-hot speaker encoder with 4 speakers in total and 2 in one batch.  1) I cannot find the learning rate you used in the paper. What is the learning rate you used? 
2) The decoder in the paper is slightly different from your code (in model_vc.py) here. Which one is correct?
3) You mentioned that training data has 100k steps. Is it the dimension of the frames of the mel-spectrogram? So the shape of each batch data sample is (batch, 1, 100*1024, 80)?
Thank you!

**Comment by auspicious3000** [comment]
The complete reproducible training code may be sent through email upon special request for non-commercial purposes.
